syntax = "proto3";

package data_node;

service DataNode {
    rpc SendFile(stream Chunk) returns (Reply);
}

message Chunk {
    bytes buffer = 1;
}

message Reply {
    int32 length = 1;
}